pinot-core/src/main/java/org/apache/pinot/core/query/pruner/ColumnValueSegmentPruner.java

```diff
@@ -251,6 +251,7 @@ private boolean pruneInPredicate(IndexSegment segment, InPredicate inPredicate,
     assert dataSource != null;
     DataSourceMetadata dataSourceMetadata = dataSource.getDataSourceMetadata();
     List<String> values = inPredicate.getValues();
+    BloomFilterReader bloomFilter = dataSource.getBloomFilter();
```

Comment:
(nit) Move this line to line 268 for readability

Reply:
done!